src/cpp/include/openvino/genai/generation_handle.hpp

```diff
@@ -88,10 +91,19 @@ class OPENVINO_GENAI_EXPORTS GenerationHandleImpl {
     GenerationStatus get_status();
 
     bool can_read();
-    bool is_dropped();
 
+    bool is_stopped();
+
+    bool is_cancelled();
+
+    OPENVINO_DEPRECATED("Please, use `stop()` instead of `drop()`.")
     void drop();
 
+    void stop();
+
+    void cancel();
+
+    GenerationOutputs back();
```

Comment:
please, drop this method 

it was explicitly removed in another PR

Reply:
removed